### PR TITLE
chore: hide stable released feature flags from admin menu

### PR DESCRIPTION
**Pull request by MounirDhahri**
The type of this PR is: **TYPE**

### Description

This PR:
- hides stable released feature flags from the admin menu
- removes the alphabetical sorting from the feature flags list which was making it hard to find newly added feature flags 

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Hide stable released feature flags from admin menu - mounir

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Hide stable released feature flags from admin menu - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d51b0cccc278195224ac6a22c300f0c52c941642
</p>

**Review by gkartalis — APPROVED** [review]
Great idea 🚀

File: src/lib/utils/AdminMenu.tsx [hunk 1]
```diff
@@ -40,10 +40,9 @@ import { getBuildNumber, getUniqueId, getVersion } from "react-native-device-inf
 import Keychain from "react-native-keychain"
 import { useScreenDimensions } from "./useScreenDimensions"
 
-const configurableFeatureFlagKeys = sortBy(
-  Object.entries(features).filter(([_, { showInAdminMenu }]) => showInAdminMenu),
-  ([k, { description }]) => description ?? k
```
Comment (pvinis) [inline src/lib/utils/AdminMenu.tsx:45]: why did we remove the sorting here? is it to have them in order of addition or some other reason?

**Review by pvinis — APPROVED** [review]
great! that makes it cleaner.

File: src/lib/utils/AdminMenu.tsx (hunk 1 above)
Comment (pvinis) [reply src/lib/utils/AdminMenu.tsx:45]: i read that on your PR desc. nice!

**Pull request merged**